src/pages/data-integrity/use-data-integrity-checks.js

```diff
@@ -0,0 +1,16 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+
+const dataIntegrityChecksQuery = {
```

Comment:
I'd move this query/hhook with `useDataIntegritySummary` to be consistent with the details hook and to make sure this is not consumed directly without the summary too, and totally abstract the backend implementation since this is all a workaround for the API design from FE perspective